technical-reports/color/gradient.md

```diff
@@ -0,0 +1,134 @@
+# Gradient
```

Comment:
- [ ] Remove this file as it’s a separate proposal (but this can be a new PR if we want to review)
- [ ] Changes instead should go in `composite-types.md`—updating all those examples to reference valid colors